sim/battle.ts

```diff
@@ -2492,16 +2488,32 @@ export class Battle extends Dex.ModdedDex {
 			}
 		}
 
-		if (!this.p1.pokemonLeft && !this.p2.pokemonLeft) {
+		let team1PokemonLeft = this.sides[0].pokemonLeft;
+		let team2PokemonLeft = this.sides[1].pokemonLeft;
+		const team3PokemonLeft = this.gameType === 'free-for-all' && this.sides[2].pokemonLeft;
+		const team4PokemonLeft = this.gameType === 'free-for-all' && this.sides[3].pokemonLeft;
+		if (this.gameType === 'multi') {
+			team1PokemonLeft = this.sides.reduce((total, side) => total + (side.n % 2 === 0 ? side.pokemonLeft : 0), 0);
+			team2PokemonLeft = this.sides.reduce((total, side) => total + (side.n % 2 === 1 ? side.pokemonLeft : 0), 0);
```

Comment:
Am I missing something, or can't you just write

    team1PokemonLeft += team3PokemonLeft;
    team2PokemonLeft += team4PokemonLeft;

Although either way I fail to see how the battle ends when team 3 or 4 is the only one with Pokémon left.

Reply:
Oh, my fault, I misread that, those values are only set for free-for-all battles. I still think it would be less confusing to write

    team1PokemonLeft += this.sides[2].pokemonLeft;
    tem2PokemonLeft += this.sides[3].pokemonLeft;